### PR TITLE
Small UI changes

### PR DESCRIPTION
**Pull request by cybear**
The build wouldn't run because of a nullish coalescing operator. I tried adding support for this into babel.js but no such luck - there's maybe some way for Storybook to pick up the babel config that I don't know of. But since it's only this one place, maybe it's fine to just go for an old school approach for now?

## I think it looks better to vertically align the pawn in the text
Screenshot 2020-12-13 at 07 22 16

## Use big primary buttons wherever applicable (imo)
Screenshot 2020-12-13 at 07 21 58

Screenshot 2020-12-13 at 07 22 25

## Make the welcome screen background more opaque, add some margins (because they fit and look god) [3]
Screenshot 2020-12-13 at 07 21 39

### CODE REVIEW

**Comment by krawaller** [comment]
@cybear is in the house and it is awesome!! ❤️

> broke the build

Do you mean the storybook?

> primary button

This is my one concern - won't that look a bit wonky when more than one command is available? See for example Coffee.

Perhaps only the end turn button should be primary? That works sort of like a "submit" button, so it could be a nice way to highlight that!

**Comment by cybear** [comment]
> @cybear is in the house and it is awesome!! ❤️

🙌 

> > broke the build
> 
> Do you mean the storybook?

Yes!

> > primary button
> 
> This is my one concern - won't that look a bit wonky when more than one command is available? See for example Coffee.

Yeah while I like the idea with inlining the buttons in text to give a more natural flow I think they could also be separated. How about this idea then?

image
Screenshot 2020-12-13 at 09 54 01


So essentially keeping the text with inline buttons, but ALSO gathering some large buttons, always at the same place, big and visible, neat and tidy.


> Perhaps only the end turn button should be primary? That works sort of like a "submit" button, so it could be a nice way to highlight that!

Yes I am thinking about that in the same way like my proposal, to put it on a separate line. Which one do you think is a better solution?

**Comment by krawaller** [comment]
> Yeah while I like the idea with inlining the buttons in text to give a more natural flow I think they could also be separated. 

Did you see that you can swap between those two things in the UI? The default is the dynamic instruction with buttons inline:

image

But if you flick off the "Text" switch then you get static controls with just the buttons:

image

This mode is pretty much what you described - buttons always in the same place, nice and tidy.

**Comment by cybear** [comment]
After this discussion I think I'll revert the button tweaks. I understand that the "non-text" UI is what I'm after. Possibly it would help me to see both the "text UI" **and** the non-text buttons at the same time, when in that mode. But it's up for discussion and I clearly need to think more about this. 
With regards to the Storybook build failing my goal was to make a GitHub workflow that builds each project so that it's visible on a PR if it breaks. But we'll see how it goes with #419...

**Comment by krawaller** [comment]
Yeah, skipping the nullish coalesce makes total sense! That snuck in very recently, and I hadn't noticed it broke the Storybook. Very much want to keep that operational, I use it frequently, so this was a silly mistake.

File: modules/content/material/chunks/newbie/chunk.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 id: "newbie"
 ---
 
-Hey, happy to have you! Come on in, {ABOUT:id=overview,text=learn more} about the app and the {ABOUT:id=whichgames,text=type of games} you can play here.
+<b>Hey, happy to have you!</b> Come on in, {ABOUT:id=overview,text=learn more} about the app and the {ABOUT:id=whichgames,text=type of games} you can play here.
```
Comment (krawaller) [inline modules/content/material/chunks/newbie/chunk.md:5]: Can use Markdown syntax here!

```suggestion
**Hey, happy to have you!** Come on in, {ABOUT:id=overview,text=learn more} about the app and the {ABOUT:id=whichgames,text=type of games} you can play here.
```

**Pull request closed without merging**